src/filters/SelectFilterOptions/SelectFilterOptions.tsx

```diff
@@ -0,0 +1,92 @@
+import React, { ChangeEvent, Dispatch, FormEvent, FunctionComponent } from "react";
+
+import styled from "@emotion/styled";
+import { CheckboxProps, Form, InputOnChangeData } from "semantic-ui-react";
+
+import { FilterState } from "../reducer";
+import isSubstring from "../../utils/isSubstring";
+
+const OptionQueryInput = styled(Form.Input)({
+  paddingRight: ".8em",
+});
+OptionQueryInput.displayName = "OptionQueryInput";
+
+/**The type of of a filter optin. */
+interface FilterOption {
+  /**The name of the filter option. E.g. A Council Commitee's name is the id of the commitee. */
+  name: string;
+  /**The visible text of filter option. */
+  text: string;
+  /**Whether the filter option is disabled. */
+  disabled: boolean;
```

Comment:
When would a filter option be disabled?

Reply:
When the user types a search string to narrow down the list of options. The unmatched options are disabled.